plugins/processors/timestamp/README.md

```diff
@@ -0,0 +1,20 @@
+# Timestamp Processor Plugin
+
+Use the `timestamp` processor to add a unix nano timestamp to the metric.
+
+This can be used to mimic logs from syslog that you'd want to display in Chronograf.
+
+### Configuration
+
+```toml
+[[processors.timestamp]]
+  ## New tag to create
+  field_key = "timestamp"
+```
+
+### Example
+
+```diff
+- syslog,appname=myapp,facility=user,hostname=test,severity=notice message="notice msg",severity_code=5i,version=1i,facility_code=1i 1564997347582799644
++ syslog,appname=myapp,facility=user,hostname=test,severity=notice message="notice msg",severity_code=5i,version=1i,facility_code=1i,timestamp=1564997347582799644i 1564997347582799644
```

Comment:
I'm not sure I understand the use case or value of adding a tag with the timestamp of the metric. Were you able to use a parser with the syslog input?

Reply:
I believe this is for compatibility with the Chronograf Log Viewer, I just checked and without the timestamp field it is reporting the log at the unix epoch.  @mmriis Is this what you are trying to work around?

Reply:
@danielnelson exactly...

Maybe the better approach would be a PR to Chronograf which defaults to using the metric time if a timestamp tag isn't present. But TS isn't my forte...

Reply:
>Maybe the better approach would be a PR to Chronograf which defaults to using the metric time if a timestamp tag isn't present.

I would agree as that does seem to be where the issue really lies. If not a PR, at least a bug report linking to this maybe?